src/pages/footer.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Avatar, Box,  Stack, Typography, Link } from '@mui/material';
+import {
+  FaFacebookSquare as FacebookLogo,
+  FaTwitter as TwitterLogo,
+  FaInstagram as InstagramLogo,
+} from 'react-icons/fa';
+
+const textLinks = [
+  {
+    title: 'Features',
+    bodyText: [
+      {
+        text: 'Link GitHub',
+        path: 'https://github.com/proga-react-practice/kappa-team-project',
+      },
+
+    ],
+  },
+  {
+    title: 'Resources',
+    bodyText: [
+      {
+        text: 'MUI',
+        path: '',
+      },
+      {
+        text: 'Developers',
+        path: '',
+      },
+      {
+        text: 'Support',
+        path: '',
+      },
+    ],
```

Comment:
Can you create a `coming soon` page for routes that are not exist for now?

Reply:
Yes, all other pages are in progress